policy/modules/services/node_exporter.te

```diff
@@ -0,0 +1,65 @@
+policy_module(node_exporter, 1.0.0)
+
+type node_exporter_t;
+type node_exporter_exec_t;
+init_daemon_domain(node_exporter_t, node_exporter_exec_t)
+
+type node_exporter_runtime_t;
+files_runtime_file(node_exporter_runtime_t)
+
+type node_exporter_var_lib_t;
+files_type(node_exporter_var_lib_t)
+
+type node_exporter_var_log_t;
+logging_log_file(node_exporter_var_log_t)
+
+allow node_exporter_t self:fifo_file rw_fifo_file_perms;
+allow node_exporter_t self:process { getsched signal };
+allow node_exporter_t self:netlink_route_socket r_netlink_socket_perms;
+allow node_exporter_t self:tcp_socket create_stream_socket_perms;
+allow node_exporter_t self:udp_socket create_stream_socket_perms;
+
+manage_files_pattern(node_exporter_t, node_exporter_runtime_t, node_exporter_runtime_t)
+files_runtime_filetrans(node_exporter_t, node_exporter_runtime_t, file)
+
+manage_dirs_pattern(node_exporter_t, node_exporter_var_lib_t, node_exporter_var_lib_t)
+manage_files_pattern(node_exporter_t, node_exporter_var_lib_t, node_exporter_var_lib_t)
+files_var_lib_filetrans(node_exporter_t, node_exporter_var_lib_t, { dir file })
+
+append_files_pattern(node_exporter_t, node_exporter_var_log_t, node_exporter_var_log_t)
+create_files_pattern(node_exporter_t, node_exporter_var_log_t, node_exporter_var_log_t)
+setattr_files_pattern(node_exporter_t, node_exporter_var_log_t, node_exporter_var_log_t)
+logging_log_filetrans(node_exporter_t, node_exporter_var_log_t, { dir file })
+
+corenet_tcp_bind_all_ports(node_exporter_t)
+corenet_tcp_bind_generic_node(node_exporter_t)
+
+dev_read_sysfs(node_exporter_t)
+
+fs_getattr_dos_fs(node_exporter_t)
+fs_getattr_rpc_pipefs(node_exporter_t)
+fs_getattr_tmpfs(node_exporter_t)
+fs_getattr_xattr_fs(node_exporter_t)
+
+init_read_state(node_exporter_t)
+
+# Search kernel sysctl rpc attributes
+kernel_read_rpc_sysctls(node_exporter_t)
```

Comment:
Is this access required only when also using the RPC service? If so I think this should be moved into the below optional block so that this access isn't enabled when that service isn't in use.